apps/mark-scan/frontend/src/pages/empty_ballot_box_page.test.tsx

```diff
@@ -0,0 +1,40 @@
+import userEvent from '@testing-library/user-event';
+import { electionGeneralDefinition } from '@votingworks/fixtures';
+import { render } from '../../test/test_utils';
+import { createApiMock, ApiMock } from '../../test/helpers/mock_api_client';
+import { screen } from '../../test/react_testing_library';
+import { EmptyBallotBoxPage } from './empty_ballot_box_page';
+import {
+  fakeCardlessVoterLoggedInAuth,
+  fakePollWorkerAuth,
+} from '../../test/helpers/fake_auth';
+
+let apiMock: ApiMock;
+beforeEach(() => {
+  apiMock = createApiMock();
+});
+
+afterEach(() => {
+  apiMock.mockApiClient.assertComplete();
+});
+
+test('requires poll worker auth', () => {
+  const electionDefinition = electionGeneralDefinition;
+  const authStatus = fakeCardlessVoterLoggedInAuth(electionDefinition);
+  render(<EmptyBallotBoxPage authStatus={authStatus} />, {
+    apiMock,
+  });
+
+  userEvent.click(screen.getByText('Insert a poll worker card to continue.'));
```

Comment:
Why click on this?

Reply:
Typo fixed in [754ca8d](https://github.com/votingworks/vxsuite/pull/4592/commits/754ca8d15e92ae7613d047874462024d9d47bfb0)